modules/ROOT/pages/deploy-on-premises.adoc

```diff
@@ -309,6 +309,21 @@ mule -M-Dmule.deployment.parallel
 
 After performing these steps, Mule deploys in parallel all applications in the `/apps` directory.
 
+== Undeploying Applications
```

Comment:
According to this page (https://confluence.internal.salesforce.com/display/MTDT/Edit+Tips+Archive), we should only use gerunds in H1 and imperative in other headings.